.github/workflows/style_check.yml

```diff
@@ -0,0 +1,38 @@
+name: Style check
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  # Trigger on pull requests.
+  pull_request:
+
+  # Trigger on pushes master
+  push:
+    branches:
+      - "master"
+
+  # Trigger on request.
+  workflow_dispatch:
+
+jobs:
+  clang-tidy:
+    name: Check C++ Code Style
+    runs-on: ubuntu-latest
+    container:
+      image: glotzerlab/ci:2021.11-clang10_py38
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          submodules: true
+      - name: Install pre-commit
+        run: python3 -m pip install pre-commit
+      - name: Build freud and export compile commands
+        run: |
+          python3 -m pip install scikit-build
```

Comment:
Is this only installing scikit-build because it's not in the CI image but the other dependencies (Cython, TBB) are already available?

Reply:
Yes.

Reply:
A comment indicating that would be beneficial.